pkg/kopia/command/common_test.go

```diff
@@ -0,0 +1,134 @@
+// Copyright 2024 The Kanister Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package command
+
+import (
+	"os"
+
+	"gopkg.in/check.v1"
+)
+
+type CommonUtilsSuite struct{}
+
+var _ = check.Suite(&CommonUtilsSuite{})
+
+func (s *CommonUtilsSuite) TestCommonArgs(c *check.C) {
```

Comment:
These tests do not need to use `check.v1` and could be built using the support in the plain Go `testing` package.

Since this is (was) a new test, wouldn't it make sense to avoid `check.v1` here?